x-pack/plugins/stack_alerts/public/alert_types/geo_containment/query_builder/expressions/entity_index_expression.tsx

```diff
@@ -155,7 +155,7 @@ export const EntityIndexExpression: FunctionComponent<Props> = ({
     <ExpressionWithPopover
       isInvalid={isInvalid}
       value={indexPattern.title}
-      defaultValue={'Select an index pattern and geo shape/point field'}
+      defaultValue={'Select an index pattern and geo point field'}
```

Comment:
this should be i18n message. Plus how about moving to common/i18n file to avoid duplicate message